depot.json

```diff
@@ -0,0 +1 @@
+{"id":"995d7q59th"}
```

Comment:
this could be provided in env/as an action input; however since this is needed in the context of a `pull_request` trigger, secrets (and [non-sensitive variables](https://github.com/orgs/community/discussions/44322)?!) cannot be accessed

Reply:
what does this do?

Reply:
depot needs the project ID when building/baking an image